app/controllers/admin/amendments_controller.rb

```diff
@@ -19,6 +19,16 @@ def create
 
   private
 
+  def load_claim
+    @claim = Claim.find(params[:claim_id])
+  end
+
+  def ensure_claim_is_amendable
+    unless @claim.amendable?
+      render "not_amendable"
```

Comment:
Our error messages aren't actually that noticeable, and I worried that it might end up looking like the claim was amended successfully. I think it's best to either render the form that the user was trying to submit, with error messages (which doesn't work in the case where the claim has been anonymised), or otherwise to show a completely different view, which seemed like the simplest. I don't think this extra little template adds much maintenance overhead, what do you think?

Reply:
Ah, OK, that's fair enough 👍